app/experimenter/experiments/migrations/0224_analysis_add_basis.py

```diff
@@ -0,0 +1,43 @@
+# Generated by Django 3.2.15 on 2022-10-13 17:50
+
+import copy
+
+from django.db import migrations
+
+
+def analysis_add_basis(apps, schema_editor):
```

Comment:
Wait shouldn't this have been done in a previous commit?

Reply:
Sure, that should work. Right now (without this PR), the `v2` schema is optional and not used. This PR would make it required and used, so I included the data migration here to enforce it. Doesn't hurt to split out the data migration first though, I can do that.